utils/config/config.go

```diff
@@ -415,45 +415,45 @@ func (artifactoryDetails *ArtifactoryDetails) SshAuthHeaderSet() bool {
 	return len(artifactoryDetails.SshAuthHeaders) > 0
 }
 
-func (artifactoryDetails *ArtifactoryDetails) CreateArtAuthConfig() (auth.CommonDetails, error) {
+func (artifactoryDetails *ArtifactoryDetails) CreateArtAuthConfig() (auth.ServiceDetails, error) {
 	artAuth := artifactoryAuth.NewArtifactoryDetails()
 	artAuth.SetUrl(artifactoryDetails.Url)
 	return artifactoryDetails.createArtAuthConfig(artAuth)
 }
 
-func (artifactoryDetails *ArtifactoryDetails) CreateDistAuthConfig() (auth.CommonDetails, error) {
+func (artifactoryDetails *ArtifactoryDetails) CreateDistAuthConfig() (auth.ServiceDetails, error) {
 	artAuth := distributionAuth.NewDistributionDetails()
 	artAuth.SetUrl(artifactoryDetails.DistributionUrl)
 	return artifactoryDetails.createArtAuthConfig(artAuth)
 }
 
-func (artifactoryDetails *ArtifactoryDetails) createArtAuthConfig(commonDetails auth.CommonDetails) (auth.CommonDetails, error) {
-	commonDetails.SetSshUrl(artifactoryDetails.SshUrl)
-	commonDetails.SetSshAuthHeaders(artifactoryDetails.SshAuthHeaders)
-	commonDetails.SetAccessToken(artifactoryDetails.AccessToken)
+func (artifactoryDetails *ArtifactoryDetails) createArtAuthConfig(ServiceDetails auth.ServiceDetails) (auth.ServiceDetails, error) {
```

Comment:
I suggest changing the function argument to start with a lower case letter, to follow our conventions.